fail2ban/tests/sockettestcase.py

```diff
@@ -83,3 +90,17 @@ def testSocketForce(self):
 		self.server.stop()
 		serverThread.join(1)
 		self.assertFalse(os.path.exists(self.sock_name))
+
+
+class ClientMisc(unittest.TestCase):
+
+	def testPrintFormattedAndWiki(self):
+		# redirect stdout to devnull
+		saved_stdout = sys.stdout
+		sys.stdout = open(os.devnull, 'w')
+		try:
+			protocol.printFormatted()
+			protocol.printWiki()
+		finally:
+			# restore stdout
+			sys.stdout = saved_stdout
```

Comment:
3 lines of code? as wrapper? OK with try-finally are 5 lines. 
But anyway, if make this, then as decorator...